shared/chat/conversation/messages/attachment/shared.tsx

```diff
@@ -27,3 +28,46 @@ export const ShowToastAfterSaving = isMobile
       ) : null
     }
   : () => null
+
+export const GetEditStyle = (isEditing: boolean, isHighlighted?: boolean) => {
+  if (isHighlighted) {
+    return Styles.collapseStyles([styles.sent, styles.highlighted])
+  }
+  return isEditing ? styles.sentEditing : styles.sent
+}
+
+const editing = {
+  backgroundColor: Styles.globalColors.yellowLight,
+  borderRadius: 2,
+  color: Styles.globalColors.blackOrBlack,
+  paddingLeft: Styles.globalMargins.tiny,
+  paddingRight: Styles.globalMargins.tiny,
+}
+const sent = Styles.platformStyles({
+  isElectron: {
+    // Make text selectable. On mobile we implement that differently.
+    cursor: 'text',
+    userSelect: 'text',
+    whiteSpace: 'pre-wrap',
+    width: '100%',
+    wordBreak: 'break-word',
+  } as const,
+  isMobile: {
+    ...Styles.globalStyles.flexBoxColumn,
+  },
+})
+const sentEditing = {
+  ...sent,
+  ...editing,
+}
+const styles = Styles.styleSheetCreate(
+  () =>
+    ({
+      editing,
+      highlighted: {
+        color: Styles.globalColors.blackOrBlack,
+      },
+      sent,
+      sentEditing,
+    } as const)
+)
```

Comment:
It looks like these styles are the same as `chat/conversation/messages/text/index.tsx`. WDYT combine them into a shared file in `chat/conversation/messages`?

Reply:
sounds good